src/collective/lazysizes/transform.py

```diff
@@ -16,7 +16,7 @@
 ROOT_SELECTOR = '//*[@id="content"]'
 
 # elements by CSS class; http://stackoverflow.com/a/1604480
-CLASS_SELECTOR = '//*[contains(concat(" ", normalize-space(@class), " "), " {0} ")]'
+CLASS_SELECTOR = '//*[contains(@class, "{0}")]'
```

Comment:
this change needs tests.

Reply:
I'm currently working on it

Reply:
you are not considering when there are more than one class in the same element..

please leave it as it was before.

Reply:
ah - i understand the issue. good to know for the exact match trick, i always used just ``contains``.

i had a problem with not filtering the blacklisted classes, so i provided this "fix". now i have to re-investigate that problem.

Reply:
But it should be filtered with the root selector. It should read:

```CLASS_SELECTOR = ROOT_SELECTOR + '//*[contains(concat(" ", normalize-space(@class), " "), " {0} ")]```

Reply:
Of course, you're already doing it.